Deployment/src/APIM/Modules/ExternalNotifcationService/main.tf

```diff
@@ -0,0 +1,212 @@
+﻿data "azurerm_api_management" "apim_instance" {
+  resource_group_name = var.resource_group_name
+  name                = var.apim_service_name
+}
+
+# Create apim group
+resource "azurerm_api_management_group" "ens_management_group" {
+  resource_group_name = data.azurerm_api_management.apim_instance.resource_group_name
+  api_management_name = data.azurerm_api_management.apim_instance.name
+  name                = "ens-group-${local.formatted_env}"
+  display_name        = "External Notification Service Group ${var.env_suffix}"
+  description         = "Management group for users with access to the ${var.env} External Notifcation Service."
```

Comment:
Minor typo in 'Notification'

Reply:
Also in folder name

Reply:
Fixed the typo in description and folder name.